StripePaymentSheet/StripePaymentSheet/Source/PaymentSheet/PaymentSheetAppearance.swift

```diff
@@ -237,6 +237,10 @@ public extension PaymentSheet {
             /// The shadow of the primary button
             /// - Note: If `nil`, `appearance.shadow` will be used as the primary button shadow
             public var shadow: Shadow?
+
+            /// The height of the primary button
+            public var height: CGFloat = 44
```

Comment:
Let's assert this is over 0 (or some other minimum) https://github.com/stripe/stripe-ios/blob/2b3d9e62ced5b486652b244b86df39b3b9f95d52/StripePaymentSheet/StripePaymentSheet/Source/PaymentSheet/PaymentSheetAppearance.swift#L72